Sources/StreamChat/WebSocketClient/Events/NotificationEvents.swift

```diff
@@ -416,3 +416,35 @@ class NotificationInviteRejectedEventDTO: EventDTO {
         )
     }
 }
+
+/// Triggered when the current user rejects an invite to a channel.
+public struct NotificationChannelDeletedEvent: ChannelSpecificEvent {
+    let cid: ChannelId
+    let channel: ChatChannel
+    let createdAt: Date
```

Comment:
Is this createdAt property the date in which the channel got deleted? if that's the case, can we document it?